pom.xml

```diff
@@ -17,8 +17,8 @@
     <version>1.3.0-SNAPSHOT</version>
 
     <properties>
-        <asyncHttpClientVersion>1.14-MULE-009-SNAPSHOT</asyncHttpClientVersion>
-        <grizzlyVersion>2.3.36-MULE-008-SNAPSHOT</grizzlyVersion>
+        <asyncHttpClientVersion>1.14-MULE-010-SNAPSHOT</asyncHttpClientVersion>
```

Comment:
Move to the latest releases instead. We don't have any changes in the snapshots and it will only cause confusion.